README.md

```diff
@@ -41,6 +41,10 @@ For other platforms or for a more advanced setup, see also manual installation b
 
 -   [Bare metal installation](docs/setup/bare_metal.md)
 
+### Upgrade
+
+Upgrading to a newer version? See [Upgrading the eWallet Server](docs/setup/upgrading).
```

Comment:
Maybe `the eWallet server application?`

Reply:
I try to call it "OmiseGO eWallet Server" everywhere if possible.

Reply:
... and Suite...

Hence I'm going to create a new page called Glossary to pin this down for good.

Reply:
@T-Dnzt @sirn I've just added a glossary section. Please check a98169a.

Reply:
👍